solutions/mysite/index.html

```diff
@@ -0,0 +1,14 @@
+<!DOCTYPE html>
+<html>
+    <head>
+        <title> Mysite</title>
+       <link rel="stylesheet" href = "style.css">
+    </head>
+    <body>
+    <img src = "https://upload.wikimedia.org/wikipedia/commons/5/5b/Ultraviolet_image_of_the_Cygnus_Loop_Nebula_crop.jpg"></img>
```

Comment:
`<img>` tags are self-closing. Since they don't have content (anything between opening and closing tag), they don't have a closing tag.

so instead of: `<img src="..."></img>` do `<img src="..." />`